### PR TITLE
external/mbedtls: apply SECLINK changes in mbedtls alt

### PR DESCRIPTION
**Pull request by an4967**
- apply SECLINK changes in mbedtls alt

### CODE REVIEW

**Review by pillip8282 — APPROVED** [review]
lgtm

File: external/include/mbedtls/alt/common.h [hunk 1]
```diff
@@ -25,8 +25,8 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define ECP_KEY_INDEX (1)
-#define RSA_KEY_INDEX (2)
+#define ECP_KEY_INDEX (32)
+#define RSA_KEY_INDEX (32)
```
Comment (davidfather) [inline external/include/mbedtls/alt/common.h:29]: You need to explain why ECP and RSA have the same index here.

Comment (an4967) [reply external/include/mbedtls/alt/common.h:29]: @davidfather 
In SE APIs, the index is divided into ROM & RAM areas.
The ROM area uses 0~31 index, the RAM area does 31~63. Hence, the stare index should be 32 for using ROM area.
Also, SE APIs can avoid duplicated index by checking empty slot, so the key indexs are okay even starts from same number.

**Review by davidfather — APPROVED** [review]
Looks good to me.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**